hyperspy/docstrings/signal.py

```diff
@@ -28,3 +28,10 @@
 
        -------------------- {0} --------------------
         {1} """
+
+OPTIMIZE_ARG = \
+    """optimize : bool, default is False
+            If the data should be re-ordered in memory, most likely making a
```

Comment:
What about something like this:

If `True`, the location of the data in memory is optimised for the fastest iteration over the navigation axes. This operation can cause a peak of memory usage and require considerable procession times for large datasets and/or low specification hardware.